_pyceres/core/problem.h

```diff
@@ -270,7 +270,7 @@ void BindProblem(py::module& m) {
             self.Evaluate(options, nullptr, &residuals, nullptr, nullptr);
             return residuals;
           },
-          py::arg("options") = ceres::Problem::EvaluateOptions())
+          py::arg_v("options", "EvaluateOptions()"))
```

Comment:
Why removing the default value?

Reply:
pybind11_stubgen generates `<EvaluateOptions...>` code otherwise. We use a similar workaround in pycolmap at the moment for default arguments that result to bindings.

Reply:
I understand, but I think that the syntax should be `py::arg_v("options", ceres::Problem::EvaluateOptions(), "EvaluateOptions()")` (it needs the default value as second argument and the displayed value as third).

Reply:
Yes, you are right, thanks.